src/services/study/index.js

```diff
@@ -313,3 +313,20 @@ export function getServersInfos() {
         return reason;
     });
 }
+
+export function fetchAvailableFilterEnumValues(
+    studyUuid,
+    nodeUuid,
+    computingType,
+    filterEnum
+) {
+    console.info('fetch available filter values');
```

Comment:
This log could provide some more info (computingType and filterEnum). (Personally I don't find those very useful but I can see they are everywhere so I guess this is a gridsuite convention)

Reply:
okey